.nvmrc

```diff
@@ -1 +1 @@
-22
+20
```

Comment:
was this intentional?

Reply:
yes, gets rid of the punycode warning 🤣

Reply:
ooh that is exciting. i really hate that warning.